components/test_raftstore/src/pd.rs

```diff
@@ -803,8 +803,8 @@ impl TestPdClient {
     }
 
     pub fn must_have_peer(&self, region_id: u64, peer: metapb::Peer) {
-        for _ in 1..500 {
-            sleep_ms(10);
+        for retry in 1..500 {
+            sleep_ms(10 * retry);
```

Comment:
The total timeout is 20mins.😂

Reply:
I think I can reduce the retry times to 100 or so, and it won't exceed 1min

Reply:
Is it the problem of retry time? I think it's scary to run a test case for more than 5 seconds already.

Reply:
@BusyJay According to the log of the latest report in #7570, it finishes applying snapshot just a little more than 5 seconds after starting adding the peer.